CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 
 [![PyPI version](https://badge.fury.io/py/iotedgehubdev.svg)](https://badge.fury.io/py/iotedgehubdev)
+## 0.12.0 - 2019-11-xx
+### Added
+* Add cert extensions required by iot edge transparent gateway
+
+### Changed
+* Add python version requirement in README and setup.py
```

Comment:
Specify here that we now only several versions of python?

Reply:
OKOK